src/app/models/Property.ts

```diff
@@ -0,0 +1,42 @@
+import { Client } from "./Client";
+import { Coordinates } from "./Coordinates";
+import { Address } from "./Address";
+
+export interface Property {
+  id?: number;
+  price?: number;
+  lost_size?: number;
```

Comment:
@Genza999, did you mean to write lotSize?

Reply:
Thanks @CryceTruly, I have changed the name